aspnetcore/signalr/groups.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Manage users and groups in SignalR
+author: rachelappel
+description: Overview of ASP.NET Core SignalR User and Group management.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/04/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/groups
+---
+
+# Manage users and groups in SignalR
+
+By [Brennan Conroy](https://github.com/BrennanConroy)
+
+Users and groups is how SignalR exposes connection management.
```

Comment:
nit: "This sounds a little weird to me. Maybe "is how" ->  "are how"

Reply:
SignalR allows messages to be sent to named groups of connections, as well as to all connections associated with a specific user.